evalai/utils/submissions.py

```diff
@@ -264,17 +264,17 @@ def display_submission_details(submission_id):
     pretty_print_submission_details(response)
 
 
-def display_submission_result(submission_id):
+def display_submission_with(submission_id, file_name, error_message):
```

Comment:
Nice idea of including error_message in`display_submission_with(submission_id, file_name, error_message)` 💯